test/test-addon.js

```diff
@@ -1,10 +1,18 @@
 'use strict'
 
 var test = require('tape')
-var execFile = require('child_process').execFile
 var path = require('path')
+var fs = require('graceful-fs')
+var child_process = require('child_process')
```

Comment:
question to @nodejs/node-gyp 
Can we use destructuring?
```js
const { execFileSync, execFile } = require('child_process')
```
Or do we test on pre ES6 versions?